mcs/class/Mono.Profiler.Log/Makefile

```diff
@@ -9,7 +9,7 @@ KEYFILE = $(LIBRARY_SNK)
 LIB_REFS = System System.Core
 LIB_MCS_FLAGS = /unsafe /publicsign /nowarn:0618
 
-XTEST_LIB_REFS = System System.Core
+XTEST_LIB_REFS = System System.Core Facades/System.Threading.Tasks
```

Comment:
What's the need for this added  facade?

Reply:
it's needed by the xunit assemblies (we already add it on all the other assemblies that run xunit tests):

```
/Users/alexander/dev/mono/external/corefx/src/CoreFx.Private.TestUtilities/src/System/AssertExtensions.cs(20,26): error CS0012: The type 'Task' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Threading.Tasks, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
```